specs/signature-specification.md

```diff
@@ -135,29 +135,31 @@ Signed attributes/claims are additional metadata apart from the payload, which a
 
-Notary requires the signature envelope to support the following signed attributes/claims.
+Notary Project requires the signature envelope to support the following signed attributes/claims.
 
 #### Standard attributes
 
-- **Signing Scheme** (critical): A REQUIRED claim that defines the [Notary Signing Scheme](./signing-scheme.md) used by the signature. This attribute dictates the rest of signature schema - the set of signed and unsigned attributes to be included in the signature. Supported values are `notary.x509` and `notary.x509.signingAuthority`.
+- **Signing Scheme** (critical): A REQUIRED claim that defines the [Notary Project Signing Scheme](./signing-scheme.md) used by the signature. This attribute dictates the rest of signature schema - the set of signed and unsigned attributes to be included in the signature. Supported values are `notary.x509` and `notary.x509.signingAuthority`.
 - **Signing Time**: A claim that indicates the time at which the signature was generated. Though this claim is signed by the signing key, it’s considered unauthenticated as a signer can modify local time and manipulate this claim. More details [here](#signing-time). This claim is REQUIRED and only valid when signing scheme is `notary.x509` .
 - **Authentic Signing Time** (critical): The authenticated time at which the signature was generated. This claim is REQUIRED and only valid when signing scheme is `notary.x509.signingAuthority` . More details [here](#signing-time).
 - **Expiry** (critical): An OPTIONAL claim that provides a “best by use” time for the artifact, as defined by the signer. More details [here](#expiry).
 - **Content Type** (critical): A REQUIRED claim that indicates the content type of the [payload](#payload). The supported value is `application/vnd.cncf.notary.payload.v1+json`. Other payload types MAY be supported in future.
 
 #### Extended attributes
 
-Implementations of Notary signature spec MAY include additional signed attributes in the signature envelope.
+Implementations of Notary Project signature spec MAY include additional signed attributes in the signature envelope.
 These attributes MAY be marked critical, i.e. the attribute MUST be understood and processed by a verifier, unknown critical attributes MUST cause signature verification to fail.
 Usage of extended signed attributes which are marked critical in signature will have implications on portability of the signature, these are discussed in [Signature Portability](#signature-portability) section.
 
 #### Extended attributes for *Notation* Plugins
 
-This section documents extended attributes used by Notary reference implementation *Notation* to support plugins.
+Notation is a CLI and set of libraries that offer an implementation of the Notary Project signature specification. Notation allows you to sign artifacts and attaching Notary Project signatures as well as verifying Notary Project signatures.
```

Comment:
There are mix of verb tenses in the second sentence. I believe grammatically correct one is:

> Notation allows you to sign artifacts and attach Notary Project signatures as well as verify Notary Project signatures.

It is better if we use simplified language, so I would propose:
> Notation allows you to sign artifacts as well as verify artifacts using Notary Project signatures.